common/h/registers/x86_64_regs.h

```diff
@@ -77,6 +77,7 @@ namespace Dyninst { namespace x86_64 {
   const signed int YMM    = 0x000B0000;  // YMM0-YMM15 Registers from AVX2/FMA (YMM0-YMM31 for AVX-512)
   const signed int ZMM    = 0x000C0000;  // ZMM0-ZMM31 Registers from AVX-512
   const signed int KMASK  = 0x000D0000;  // K0-K7 opmask Registers from AVX-512
+  const signed int FPCTL  = 0x000E0000;  // 32-bit x87 and SSE control registers
```

Comment:
both 32 and 64-bit and mxcsr is used by avx so change comment to `x87, SSE and AVX control/status registers`.

Reply:
I was looking at the wrong definition of FULL.